example/analysis_options.yaml

```diff
@@ -0,0 +1,15 @@
+include: package:solid_lints/analysis_options.yaml
+
+linter:
+  rules:
+    avoid_classes_with_only_static_members: false
```

Comment:
Would love to discuss rule changes vs standard lint.

Reply:
most of them were added in 0.0.19, I forgot to update to 0.0.19
Now there are fewer changes

example:
- `public_member_api_docs: false` - doesn't really matter for the example

lib:
- `package_api_docs:true` - not present in `solid_lints` but useful for packages

common:
- `prefer_foreach: true` - can be considered a style choice, I like it because it makes code more concise
- `prefer_single_quotes: true` - I didn't add it to `solid_lints` because it is a style choice, but I think that it is better to commit to one style instead of mixing both if possible. I chose the single quotes because the character is simpler so it produces less visual noise